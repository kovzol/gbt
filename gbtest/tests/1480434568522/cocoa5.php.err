0.00user 0.00system 0:00.01elapsed 66%CPU (0avgtext+0avgdata 10204maxresident)k
0inputs+8outputs (0major+694minor)pagefaults 0swaps
