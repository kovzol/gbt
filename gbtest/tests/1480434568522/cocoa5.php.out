*** Cannot access default package directory: "packages" ***

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!! WARNING: Standard CoCoA packages not found !!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

   ______      ______      ___         ______
  / ____/___  / ____/___  /   |       / ____/
 / /   / __ `/ /   / __ `/ /| |______/___ `  
/ /___/ /_/ / /___/ /_/ / ___ /_____/___/ /  
`____/`____/`____/`____/_/  |_|    /_____/   
With CoCoALib and external libraries CDD FROBBY GFAN GSL NORMALIZ READLINE 
indent(VersionInfo()); -- for information about this version

# use QQ[v20,v21,v19,v16,v15,v12,v11,v10,v9,v8,v7,v6,v5,v4,v3,v1,v2], DegRevLex<v12,v11,v10,v9,v8,v7,v6,v5,v4,v3,v1,v2], DegRevLex;                         

# GBasis(ideal(-1*v11,-1*v9,-1+v12,v10,-1+v7,v6,v8,v5,-1*v11*v10+v12*v9+v11*v4+<1+v12,v10,-1+v7,v6,v8,v5,-1*v11*v10+v12*v9+v11*v4+-                         1*v9*v4+-1*v12*v3+v10*v3,<1*v11*v10+v12*v9+v11*v4+-1*v9*v4+-1*v12*v3+v10*v3,-                         1*v8+v6+-1*v7*v6+v8*v5,-1<*v9*v4+-1*v12*v3+v10*v3,-1*v8+v6+-1*v7*v6+v8*v5,-1*                         v7*v6+v8*v5+v7*v4+-1*v5*v<*v8+v6+-1*v7*v6+v8*v5,-1*v7*v6+v8*v5+v7*v4+-1*v5*v4                         +-1*v8*v3+v6*v3,-1+v19^2+<7*v6+v8*v5+v7*v4+-1*v5*v4+-1*v8*v3+v6*v3,-1+v19^2+-                         1*v2^2+2*v1+-1*v1^2,v21^4<-1*v8*v3+v6*v3,-1+v19^2+-1*v2^2+2*v1+-1*v1^2,v21^4+                         -2*v21^2*v20^2+v20^4+-2*v<*v2^2+2*v1+-1*v1^2,v21^4+-2*v21^2*v20^2+v20^4+-2*v2                         1^2*v19^2+-2*v20^2*v19^2+<2*v21^2*v20^2+v20^4+-2*v21^2*v19^2+-2*v20^2*v19^2+v                         19^4,-1*v16^2+2*v15+-1*v1<^2*v19^2+-2*v20^2*v19^2+v19^4,-1*v16^2+2*v15+-1*v15                         ^2+v4^2+-2*v3+v3^2,v20^2+<9^4,-1*v16^2+2*v15+-1*v15^2+v4^2+-2*v3+v3^2,v20^2+-                         1*v4^2+-1*v3^2+2*v4*v2+-1<2+v4^2+-2*v3+v3^2,v20^2+-1*v4^2+-1*v3^2+2*v4*v2+-1*                         v2^2+2*v3*v1+-1*v1^2,1+-1<*v4^2+-1*v3^2+2*v4*v2+-1*v2^2+2*v3*v1+-1*v1^2,1+-1*                         v16^2+-1*v15^2+2*v16*v4+-<2^2+2*v3*v1+-1*v1^2,1+-1*v16^2+-1*v15^2+2*v16*v4+-2                         *v3+2*v15*v3,v21^2+-1*v16<16^2+-1*v15^2+2*v16*v4+-2*v3+2*v15*v3,v21^2+-1*v16^                         2+-1*v15^2+2*v16*v2+-1*v2<v3+2*v15*v3,v21^2+-1*v16^2+-1*v15^2+2*v16*v2+-1*v2^                         2+2*v15*v1+-1*v1^2));
[v5, v6, v7 -1, v8, -v9, v10, -v11, v12 -1, v4, -v3, -4*v16^2 +3, -2*v15 +1, v19^2 -v1^2 -v2^2 +2*v1 -1, v21^2 -v1^2 +2*v16*v2 -v2^2 +v1 -1, v20^2 -v1^2 -v2^2, -3*v1^4 +4*v16*v1^2*v2 -6*v1^2*v2^2 +4*v16*v2^3 -3*v2^4 +6*v1^3 -4*v16*v1*v2 +6*v1*v2^2 -3*v1^2 -v2^2]

# 